src/main/resources/db/migration/V1_123__update_transfer_pool_procedure.sql

```diff
@@ -0,0 +1,106 @@
+create or replace procedure juror_mod.create_bureau_snapshot(in juror_pool_rec record)
+	language plpgsql
+		as $procedure$
+
+begin
+
+	insert into juror_mod.bureau_snapshot (juror_number, pool_number, "owner", user_edtq, is_active, status, def_date, pool_seq, edit_tag, next_date, was_deferred, deferral_code, postpone, scan_code, last_update, reminder_sent, transfer_date, date_created, excusal_code, acc_exc, police_check)
+	select		jp.juror_number,
+				jp.pool_number,
+				jp."owner",
+				jp.user_edtq,
+				jp.is_active,
+				jp.status,
+				jp.def_date,
+				jp.pool_seq,
+				jp.edit_tag,
+				jp.next_date,
+				jp.was_deferred,
+				jp.deferral_code,
+				jp.postpone,
+				jp.scan_code,
+				jp.last_update,
+				jp.reminder_sent,
+				current_date,
+				jp.date_created,
+				j.excusal_code,
+				j.acc_exc,
+				j.police_check
+	from 		juror_mod.juror_pool jp
+	inner join	juror_mod.juror j
+		on		jp.juror_number = j.juror_number
+	where 		jp.juror_number = juror_pool_rec.juror_number
+				and jp.pool_number = juror_pool_rec.pool_number
+				and jp."owner" = '400';
```

Comment:
add status comparison in as per the below update statement for the scenario where a user has amended a record whilst hte pool_transfer is running